pyproject.toml

```diff
@@ -45,7 +46,7 @@ pytest-django = ">=4.1"
 pytest-cov = ">=2.10"
 pytest-isort = ">=1.2"
 pytest-flake8 = ">=1.0"
-pytest-black = ">=0.3"
+pytest-black = {version = ">=0.3", python = ">=3.6.2,<4"}
```

Comment:
What is the reason for this change?

Reply:
I think we can restrict the Python version for the whole cacheback project to at least 3.6.2. I had a similar issue in another project of mine: https://github.com/stephrdev/django-cruditor/blob/main/pyproject.toml#L26
With that, this change shouldn't be needed on the dependency entry.